packages/plugins/puppet-foreman_scap_client/puppet-foreman_scap_client.spec

```diff
@@ -30,24 +28,21 @@ cp -rp . %{buildroot}/%{puppet_foreman_scap_client_dir}/
 
 
 %files
-%doc NEWS COPYING README.md
-%dir %{puppet_modules_dir}/%{puppet_module}
-%{puppet_foreman_scap_client_dir}/COPYING
-%{puppet_foreman_scap_client_dir}/README.md
-%{puppet_foreman_scap_client_dir}/NEWS
+%doc NEWS README.md REFERENCE.md
+%dir %{puppet_foreman_scap_client_dir}
+%license %{puppet_foreman_scap_client_dir}/COPYING
```

Comment:
Bleh, I just realized this probably should have been `%license COPYING` and then `%exclude` this file so it's properly placed in `/usr/share/licenses`. Still, this isn't worse than it used to be.